Puara/halp_utils.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
Since this is in all your PRs, I'd propose you to do the following: 

1/ make a first PR & commit with just that file (and any common helper files you reuse across your PRs)
2/ rebase your PRs on top of this 

what do you think?